### ISSUE TITLE
Section 3.6 refers to `setup.py` (which does not exist)

### ISSUE THREAD

**Issue by robveijk** [pycabook/pycabook#51]
Section 3.6 mentions:
```
For the time being, create a file `cli.py` in the same directory that contains `setup.py`.
```
Since `setup.py` does not exist (yet?), it might be better to update this to something like
```
+For the time being, create a file `cli.py` in the repository root (the same directory that contains `README.md`).
```

Thanks for writing such an awesome book! :)

**Comment by lgiordani** [comment]
Hi! Thanks @robveijk for spotting it. That was meant to be `pyproject.toml`. Probably it's a leftover of a previous configuration. At any rate I will fix it in version 2.0.2 (soon out with other fixes).

**Comment by robveijk** [comment]
> Hi! Thanks @robveijk for spotting it. That was meant to be `pyproject.toml`. Probably it's a leftover of a previous configuration. At any rate I will fix it in version 2.0.2 (soon out with other fixes).

That's awesome @lgiordani . Thanks! 🙏

**Comment by lgiordani** [comment]
This has been solved in version 2.1.0. Thanks!
